grafana/dashboards.yaml

```diff
@@ -0,0 +1,4773 @@
+---
+# Source: knative/templates/knative-dashboards.yaml
```

Comment:
Just wondering if we need this file checked in - since the README mentions the config map is for testing

Reply:
added a sed command, can you lift the branch protection once more?